doc/ext/docscrape_sphinx.py

```diff
@@ -129,6 +129,8 @@ def _str_member_list(self, name):
             #     out += [''] + autosum
 
             if others:
+                out += [r'.. tabularcolumns:: p{3cm}p{\dimexpr\linewidth-3cm-4\tabcolsep\relax}']
```

Comment:
perhaps better (see [comment](https://github.com/sympy/sympy/pull/26867#issuecomment-2263662908)):
```suggestion
                out += [r'.. tabularcolumns:: p{3.5cm}p{\dimexpr\linewidth-3.5cm-4\tabcolsep\relax}']
```